packages/tupaia-web/src/features/EnlargedDashboardItem/utils/useExportDashboardItem.tsx

```diff
@@ -87,24 +87,39 @@ export const useExportDashboardItem = (
   );
 
   const { config } = currentDashboardItem || ({} as DashboardItem);
-  const { type, presentationOptions, name } = config || ({} as DashboardItemConfig);
-  const exportTitle = `${name}, ${entityName}`;
 
-  const { doExport } = useChartDataExport(
-    {
-      ...reportData,
+  const exportTitle = `${config?.name}, ${entityName}`;
+
+  const getExportViewConfig = () => {
+    if (!config || config?.type !== 'chart') return config;
+    const { chartType } = config;
+    if (chartType === 'gauge') return config;
+    const { presentationOptions } = config;
+    return {
       ...config,
       presentationOptions: {
-        ...(presentationOptions || {}),
+        ...presentationOptions,
         exportWithLabels,
         exportWithTable,
         exportWithTableDisabled,
       },
-    } as ViewContent,
-    exportTitle,
-  );
+    };
+  };
+
+  const getExportViewContent = () => {
+    if (config?.type === 'matrix' || config?.type === 'component') return undefined;
+    const exportViewConfig = getExportViewConfig();
+    return {
+      ...reportData,
+      ...exportViewConfig,
+    } as ExportViewContent; // casting here to handle the different types of data that can be passed to the ExportViewContent component, which the union type doesn't cover well, and the fact that DashboardItemReport doesn't know when type is chart that it returns a ChartData type and not a ViewDataItem type etc
```

Comment:
Not super stoked with having to cast here, but I was finding it really difficult when the data was different shapes, and the report type doesn't know that it is returning a chart report when type is chart, etc. Unsure if there is a better way?